lib/pf/UnifiedApi/Controller/Config/Certificates.pm

```diff
@@ -0,0 +1,380 @@
+package pf::UnifiedApi::Controller::Config::Certificates;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Config::Certificates - 
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Config::Certificates
+
+
+
+=cut
+
+use strict;
+use warnings;
+use pf::ssl;
+use pf::util;
+use File::Slurp qw(read_file);
+use pf::error qw(is_error);
+use Mojo::Base qw(pf::UnifiedApi::Controller::RestRoute);
+use pf::file_paths qw(
+    $server_cert
+    $server_key
+    $server_pem
+    $radius_server_cert
+    $radius_ca_cert
+    $radius_server_key
+);
+
+my $CERT_DELIMITER = "-----END CERTIFICATE-----";
+
+
+sub certs_map {
+    return {
+        http => {
+            cert_file => $server_cert,
+            key_file => $server_key,
+            bundle_file => $server_pem,
+        },
+        radius => {
+            cert_file => $radius_server_cert,
+            ca_file => $radius_ca_cert,
+            key_file => $radius_server_key,
+        },
+    };
+}
+
+=head2 resource
+
+Validate the resource
+
+=cut
+
+sub resource {
+    my ($self) = @_;
+    my $id = $self->stash->{certificate_id};
+    unless (defined($self->resource_config($id))) {
+        return $self->render_error("404", "Item ($id) not found");
+    }
+
+    return 1;
+}
+
+=head2 resource_config
+
+Get the configuration associated to a resource
+
+=cut
+
+sub resource_config {
+    my ($self, $certificate_id) = @_;
+    $certificate_id //= $self->stash->{certificate_id};
+    return $self->certs_map->{$certificate_id};
+}
+
+=head2 read_from_disk
```

Comment:
I think we should s/disk/file/ everywhere. You're reading a file afterall - it can be on a "disk", or not.

Reply:
I took your advice but renamed this to `read_from_tape_backup`

Made much more sense to me.

Also thought of `read_from_zaymes_sandwich` but base64 was too complex to be stored on that given @lzammit considers base64 as encryption

Reply:
I would have preferred read_from_floppy_disk as it's likely the most secured medium to store data now because no-one can read them anymore.

Reply:
We could also store this on an 8-track and use an Oldsmobile as the interface to read the certs from

Reply:
We should also encode/decode using ROT26